consul/src/main/scala/io/buoyant/consul/CatalogNamer.scala

```diff
@@ -51,11 +51,11 @@ class CatalogNamer(
     def get(name: String): DcCache = synchronized {
       activity.sample() match {
         case Activity.Ok(snap) =>
-          snap.get(name).getOrElse {
+          snap.getOrElse(name, {
```

Comment:
yeah, i tend to prefer explicit match statements, especially in synchronized blocks.